src/gemini_extractor.py

```
@@ -0,0 +1,547 @@
+            # Add content
+            if section.content and section.content != section.title:
+                lines.append(section.content)
+
+            lines.append("")  # Empty line
+
+        return "\n".join(lines)
+
+
+def get_extractor(
+    config: Optional[ExtractionConfig] = None, backend: str = "auto"
+) -> Union["GeminiExtractor", "UnstructuredExtractor"]:
```

Ruff (0.14.5)

Undefined name `UnstructuredExtractor`

(F821)